<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>379</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>359</width>
    <height>379</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>758</width>
    <height>718</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ИС Аэропорт</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/new/prefix1/icons1/samolet.jpg</normaloff>:/new/prefix1/icons1/samolet.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Добро пожаловать.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="viewRoutesButton">
      <property name="text">
       <string>Доступные рейсы</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="manageRoutesButton">
      <property name="text">
       <string>Создание / Удаление рейса</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="viewUsersButton">
      <property name="text">
       <string>Просмотр пользователей</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="viewPassesButton">
      <property name="text">
       <string>Просмотр пассажиров</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="viewRequestsButton">
      <property name="text">
       <string>Просмотр запросов</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="makeRequestButton">
      <property name="text">
       <string>Сделать запрос билета</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="myInfoButton">
      <property name="text">
       <string>Мои данные</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>359</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="about"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="exit">
   <property name="text">
    <string>Выйти</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>viewUsersButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>viewUsers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewPassesButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>viewPasses()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manageRoutesButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>manageRoutes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewRoutesButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>viewRoutes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>makeRequestButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>doRequest()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewRequestsButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>viewTickets()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>myInfoButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>viewMyInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>about</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>tellAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>viewUsers()</slot>
  <slot>viewPasses()</slot>
  <slot>manageRoutes()</slot>
  <slot>viewRoutes()</slot>
  <slot>doRequest()</slot>
  <slot>viewTickets()</slot>
  <slot>quit()</slot>
  <slot>viewMyInfo()</slot>
  <slot>tellAbout()</slot>
 </slots>
</ui>
